argilla-server/src/argilla_server/api/schemas/v1/records.py

```diff
@@ -82,13 +83,20 @@ class Config:
 
 
 class RecordCreate(BaseModel):
-    fields: Dict[str, Union[StrictStr, None, List[Dict[StrictStr, StrictStr]]]]
+    fields: Dict[str, Union[StrictStr, None, List[ChatMessage]]]
```

Comment:
Be careful with the order here. Pydantic will parse this in order until one matches. So it's recomendable to go from more specific to less specific, so something like this:

```suggestion
    fields: Dict[str, Union[List[ChatMessage], StrictStr, None]]
```